python_src/src/lamp_py/migrations/versions/metadata_prod/07903947aabe_initial_changes.py

```diff
@@ -40,6 +45,52 @@ def upgrade() -> None:
         unique=False,
         postgresql_where=sa.text("rail_pm_processed = false"),
     )
+
+    # pull metadata from the rail performance manager database into the
+    # metadata database. the table may or may not exist, so wrap this in a try
+    # except
+    try:
+        rpm_db_manager = DatabaseManager(
+            db_index=DatabaseIndex.RAIL_PERFORMANCE_MANAGER
+        )
+        md_db_manager = DatabaseManager(db_index=DatabaseIndex.METADATA)
+
+        insert_data = []
+        # pull metadata from the rail performance manager database via direct
+        # sql query. the metadata_log table may or may not exist.
+        with rpm_db_manager.session.begin() as session:
+            result = session.execute(
+                "SELECT path, processed, process_fail FROM metadata_log"
+            )
+            for row in result:
+                (path, processed, process_fail) = row
+                insert_data.append(
+                    {
+                        "path": path,
+                        "rail_pm_processed": processed,
+                        "rail_pm_process_fail": process_fail,
+                    }
+                )
+
+        # insert data into the metadata database
+        with md_db_manager.session.begin() as session:
+            # do nothing on conflicts in file paths
+            result = session.execute(
+                psql_insert(MetadataLog.__table__)
+                .values(insert_data)
+                .on_conflict_do_nothing()
+            )
```

Comment:
i'm a little concerned in a case where this migration gets run out of sync with the migration on the production manager database and we find ourselves in a situation where we can't do anything to unblock prod since we can't manually adjust the databases. ingestion should only ever have flags marked as false, so i don't think we would be at risk of missing processing of incoming data.

happy to revert this though if you think its risky.